array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,9 @@
+/* exported includes */
+
+function includes(array, value) {
+  if (array.indexOf(value) >= 0) {
+    return true;
+  } if (array.indexOf(value) === -1) {
```

Comment:
There's no need for this second conditional here. if the first conditional isn't met, the value wasn't in the array, so you can safely just `return false`